trunk/Lib/Date.php

```diff
@@ -131,10 +129,12 @@ private static function getMonths() {
      * @return array
      */
     public static function getDaysTranslated() {
-        $currentLocale = setlocale( LC_ALL, 0 );
-        setlocale( LC_ALL, get_locale() );
         $dayList = self::getDays();
-        setlocale( LC_ALL, $currentLocale );
+
+        // translate by wordpress-lang
+        foreach ( $dayList as &$day ) {
+            $day = __( $day );
```

Comment:
same as line 106

Reply:
I think this is inevitable on this point.

Reply:
It is now hardcoded